packages/now-build-utils/src/detect-routes.ts

```diff
@@ -281,7 +281,12 @@ async function detectApiRoutes(
           continue: true,
         },
       ];
-    } else {
+    } else if (
+      defaultRoutes.some(
+        route =>
+          (route as Source).dest && (route as Source).dest!.startsWith('/api')
```

Comment:
```suggestion
          !isHandler(route) && route.dest && route.dest.startsWith('/api')
```

This should make typescript happy 🙂